airflow/providers/celery/executors/celery_executor_utils.py

```diff
@@ -298,7 +298,7 @@ def _get_many_using_multiprocessing(self, async_results) -> Mapping[str, EventBu
         num_process = min(len(async_results), self._sync_parallelism)
 
         with ProcessPoolExecutor(max_workers=num_process) as sync_pool:
-            chunksize = max(1, math.floor(math.ceil(1.0 * len(async_results) / self._sync_parallelism)))
+            chunksize = max(1, math.ceil(len(async_results) / self._sync_parallelism))
```

Comment:
I wonder why the outer `floor` existed in the first place. That wouldn’t do anything on Python 2 either…

Reply:
Exactly